configure.ac

```diff
@@ -61,6 +61,8 @@ AC_ARG_ENABLE([debug],
                              [enable debugging output])],[],[enable_debug=yes])
 AM_CONDITIONAL([ENABLE_DEBUG], test x"$enable_debug" = x"yes")
 if test x"$enable_debug" = x"yes"; then
+   CFLAGS="${CFLAGS} -O0 -g3 -fsanitize=address -fno-omit-frame-pointer -lasan -lpthread"
+   LDFLAGS="${LDFLAGS} -fsanitize=address -lasan"
```

Comment:
Can we please not make `--enable-debug` also turn on ASAN ? Other options would be to either run `./configure` in the setup script passing the appropriate `CFLAGS` and `LDFLAGS` or putting it behind a `--enable-asan` flag or having a `make asan` target that builds with those flags - all of these are fine by me, but I don't want it to be the default when you run with debugging on.